frontend/lib/resolution/disambiguation.cpp

```diff
@@ -752,6 +902,56 @@ static int compareSpecificity(const DisambiguationContext& dctx,
   }
 }
 
+/*
+ * Returns:
+ *   0 if there is no preference between them
+ *   1 if fn1 is preferred
+ *   2 if fn2 is preferred
+*/
+static int testOpArgMapping(const DisambiguationContext& dctx,
```

Comment:
This was added in production with PR #17103 but as far as I know it has never been documented in the [spec](https://chapel-lang.org/docs/language/spec/procedures.html#determining-most-specific-functions). Could you make an issue about the need to document it?

Reply:
I created https://github.com/Cray/chapel-private/issues/5637 for this